components/PostForm/form.js

```diff
@@ -46,18 +53,22 @@ const SimpleForm = props => {
       <div>
         <label>Subject</label>
         <div>
-          <Field
-            name="post_subject"
-            component="input"
-            type="text"
-            placeholder="enter a subject"
-          />
+          <NativeListener onKeyDown={handleButtonClick}>
+            <Field
+              name="post_subject"
+              component="input"
+              type="text"
+              placeholder="enter a subject"
+            />
```

Comment:
I believe you can pass a real component to the `Field` and then just pass the event as normal if it is ok. Much easier to follow and less buggy than hijacking in the long term. Something like this possibly? 
```
<Field
  name="post_subject"
  component={(props) => {
    return (
      <input 
         value={props.value} 
         {...props}
         onChange={e => {
           if(e.key > 10) {
             props.onChange(e)
           }
         }
       />
     )
  }
  type="text"
  placeholder="enter a subjecT"
/>
```

Taken from stateless function example from the docs here: https://redux-form.com/6.0.0-alpha.4/docs/api/field.md/